traitlets/config/tests/test_application.py

```diff
@@ -66,7 +66,7 @@ class MyApp(Application):
 
     aliases = Dict({
                     ('fooi', 'i') : 'Foo.i',
-                    ('j', 'fooj') : 'Foo.j',
+                    ('j', 'fooj') : ('Foo.j', "`j` terse help msg"),
```

Comment:
By the way, the number of config-params above is an order of magnitude bigger if you include `has_traits()` instead of `has_own_traits()`.  That is why my retracted 41480333b840 was trying to save space also for `--help-all`  case.